tecken/gunicornhooks.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""
+Gunicorn server hooks.
+
+See https://docs.gunicorn.org/en/stable/settings.html#server-hooks
+"""
+
+import markus
+
+from tecken import settings
+
+
+metrics = markus.get_metrics("tecken")
+
+
+def configure_markus():
```

Comment:
What is calling this function? It doesn't seem to be one of the predefined Gunicorn hooks, so I don't understand how it gets called. Maybe this should be called `post_worker_init()`, or whatever the appropriate hook is?

Reply:
Whoops--it doesn't get called. But it works when I manually test things plus the logging output is what we configure in the Django webapp and not what Gunicorn is emitting, so I wonder if `worker_abort` gets called in the gunicorn worker in the SIGABRT handling and not the Gunicorn master process.

I checked the code and that's what's going on. So we don't need the `configure_markus` and some of that other stuff at all. I'll remove it and add a note.

https://github.com/benoitc/gunicorn/blob/ca9162d9cd0f2403a4b65a54ddb996eb855e8b58/gunicorn/workers/base.py#L200-L203